### PR TITLE
Use Automatic Package Restore for nuget

### PR DESCRIPTION
**Pull request by ermshiperete**
This replaces the previously used MSBuild integrated package restore
and should enhance the build speed a little bit since it doesn't
restore the packages each time we build but instead when loading
the solution in VS/MD, or when building from the command line (using
Bloom.proj). It also allows to build when not connected to the Internet.

[Review on Reviewable](https://reviewable.io/reviews/bloombooks/bloomdesktop/795)

### CODE REVIEW

File: build/NuGet.targets [hunk 1]
```diff
@@ -25,74 +22,40 @@
 
 	<PropertyGroup>
 		<NuGetToolsPath>$(MSBuildThisFileDirectory)</NuGetToolsPath>
-		<PackagesConfig Condition="'$(OS)'=='Windows_NT'">$(ProjectDir)packages.config</PackagesConfig>
-		<PackagesConfig Condition="'$(OS)'!='Windows_NT'">$(ProjectDir)/packages.config</PackagesConfig>
```
Comment (JohnThomson) [inline build/NuGet.targets]: This change doesn't seem related...I suppose you are quite confident that the leading slash for linux is (no longer?) needed?

**Comment by ermshiperete** [comment]
It's weird what GitHub shows what happened with this file - git detected a rename, although I deleted .nuget/NuGet.targets and copied build/NuGet.targets from libpalaso/build. reviewable.io shows it correctly.


---

Review status: 0 of 8 files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/795)

**Comment by JohnThomson** [comment]
I don't see any problems, but don't know enough about nuget and how it integrates with MsBuild to be confident.

**Comment by ermshiperete** [comment]
This change implements the same way as is done in Palaso and that I'll present on Thursday.


---

Review status: 0 of 8 files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/795)
